src/docfx/build/xref/ExternalXref.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Text.Json.Serialization;
+
+namespace Microsoft.Docs.Build
+{
+    internal class ExternalXref
+    {
+        public string Uid { get; set; } = "";
```

Comment:
>"" [](start = 42, length = 2)

string.empty?

Reply:
OK.

Reply:
I tend to prefer `""` for brevity and most of the places in docfx uses `""` over `string.Empty`.

Reply:
I see. @xichan@microsoft.com , let's keep as it does

---
In reply to: [503120608](https://github.com/dotnet/docfx/pull/6631#discussion_r503120608) [](ancestors = 503120608)